src/java/install.sh

```diff
@@ -206,15 +206,15 @@ sdk_install() {
     local full_version_check=${5:-".*-[a-z]+"}
     local set_as_default=${6:-"true"}
     if [ "${requested_version}" = "none" ]; then return; fi
-    # Blank will install latest stable version SDKMAN has
-    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "lts" ] || [ "${requested_version}" = "default" ]; then
+     # Blank will install latest stable version SDKMAN has
+    if [ "${requested_version}" = "latest" ] || [ "${requested_version}" = "default" ]; then
          requested_version=""
```

Comment:
And, if this `requested_version` value is `""` (empty or blank), it automatically picks up `21.0.3-tem`, regardless the `jdkDistro` is set to `ms`.

Reply:
@gauravsaini04 Can we fix this bug as well? The Feature should consider the `jdkDistro` while computing `latest` or `lts`

Reply:
corrected